packages/mui-system/src/index.js

```diff
@@ -50,7 +50,7 @@ export { default as createStyled } from './createStyled';
 export * from './createStyled';
 export { default as styled } from './styled';
 export { default as createTheme } from './createTheme';
-export { default as createBreakpoints } from './createTheme/createBreakpoints';
+export { default as createBreakpoints } from './createBreakpoints/createBreakpoints';
```

Comment:
Do we want this in the top level export? I'd say let's keep it at path level.

Reply:
I think we need to keep it like this otherwise it's a breaking change which is out of scope of this PR.